numpy/lib/type_check.py

```diff
@@ -327,6 +327,13 @@ def nan_to_num(x):
     ----------
     x : array_like
         Input data.
+    copy : bool, optional
+        .. versionadded:: 1.13
```

Comment:
this is usually at the end of the parameter description and I think it needs a newline before the tag too.